tests/e2e/specs/setup-mc/step-4-complete-campaign.test.js

```diff
@@ -234,6 +234,54 @@ test.describe( 'Complete your campaign', () => {
 				} );
 			} );
 
+			test.describe( 'Validate budget percent', () => {
+				test.beforeAll( async () => {
+					await setupBudgetPage.mockBudgetRecommendation( {
+						currency: 'USD',
+						recommendations: [
+							{
+								country: 'US',
+								daily_budget: 100,
+							},
+						],
+					} );
+
+					await completeCampaign.goto();
+				} );
+
+				test( 'should see validation error if lower than the 30%', async () => {
+					await setupBudgetPage.fillBudget( '10' );
+					await setupBudgetPage.getBudgetInput().blur();
+
+					const error = await page
+						.locator( '.components-base-control__help' )
+						.textContent();
+					await expect( error ).toBe(
+						'Please make sure daily average cost is at least NT$30.00'
+					);
```

Comment:
Prefer `await expect( locator ).toHaveText( '...' );` to avoid flakiness.

Reply:
👍